### ISSUE TITLE
Is this a server-only package?

### ISSUE THREAD

**Issue by andrewhartwig** [anvk/google-stocks#11]
I see one of your dependencies is `request` which uses the filesystem module (from node). 

Is there any package similar that is meant for the front-end?

**Comment by anvk** [comment]
@andrewhartwig You are right I just spotted that I use `request` npm module. I guess I should've used something agnostic like `superagent` instead.

If you have `webpack` in your project I believe you should be fine. Never really researched if `request` could be used on front-end. Could you try and let me know? (once again I think it should be fine if you use something like `webpack` )

**Comment by andrewhartwig** [comment]
Actually, using create-react-app it works, but I was helping a friend debug his app, and it was failing at that part. 

I was getting CORS issues though from my front-end code.
